### ISSUE TITLE
Server should not require clients to include precompiled .digdag.yml in project archive

### ISSUE THREAD

**Issue by frsyuki** [treasure-data/digdag#47]
Now, client needs to include .digdag.yml file in a project archive file (tar.gz). But it makes it difficult to create client libraries such as ruby clients.

Idea here is that server reads digdag.yml (not .digdag.yml) as described at #45. Clients just create a tar.gz including digdag.yml and send it to server. Archiving is like this command:

```
tar -C $(basedir path/to/digdag.yml) --exclude=".*" -czvf - .
```

Client must include digdag.yml in the archive.

**Comment by frsyuki** [comment]
#322

**Comment by yoyama** [comment]
This issue looks like be resolved because #322 had been merged.
I close it but feel free to re-open.
